revolv/static/auth.css

```diff
@@ -99,3 +90,31 @@ small.field-error {
     -webkit-transition: color 0.1s; }
   .submit-button:disabled {
     opacity: 0.3; }
+  .submit-button.gray {
+    cursor: pointer;
+    background-color: transparent;
+    color: #b3b3c0;
+    border: 1px solid #b3b3c0; }
+    .submit-button.gray:hover {
+      background-color: #b3b3c0;
+      color: white;
+      transition: background-color 0.2s;
+      transition: color 0.1s;
+      -webkit-transition: background-color 0.2s;
+      -webkit-transition: color 0.1s; }
+    .submit-button.gray:disabled {
+      opacity: 0.3; }
+  .submit-button.yellow {
+    cursor: pointer;
+    background-color: transparent;
+    color: #f0ad4e;
```

Comment:
this color (#f0ad4e) is used a ton! probably the other colors too

it would probably be good to make a variable for it!